junit/src/main/java/cucumber/runtime/junit/JUnitReporter.java

```diff
@@ -6,10 +6,13 @@
 import cucumber.api.event.TestStepFinished;
 import cucumber.api.event.TestStepStarted;
 import cucumber.runner.EventBus;
+import cucumber.runtime.junit.PickleRunners.PickleRunner;
 import gherkin.pickles.PickleStep;
-import org.junit.internal.runners.model.EachTestNotifier;
+import org.junit.internal.AssumptionViolatedException;
```

Comment:
AFAIK this is the JUnit <4.12 class for the assume feature, [JUnit 4.12 use another class](https://github.com/cucumber/cucumber-jvm/pull/961), as we handle both classes in other places we should do it here to (see comment on usage).